### ISSUE TITLE
Add Android 10 Edge-to-Edge (transparent navigation bar) support.

### ISSUE THREAD

**Issue by Steve-Mr** [robertodoering/harpy#397]
Harpy has transparent navigation bar supported for android 11, but Android 10 also has similar navigation gesture as android 11.
Please consider add transparent navigation bar for android 10.
Thank you.

**Comment by robertodoering** [comment]
Hey @Steve-Mr, thanks for the suggestion.
Can you elaborate what you mean with transparent navigation bar?
If you are referring to the the gesture navigation in Android 10, that is just as supported as the gesture navigation in Android 11.

**Comment by Steve-Mr** [comment]
![Screenshot_20210729-181543](https://user-images.githubusercontent.com/24504742/127475903-7be88555-9848-439b-97c4-9c48dfb63afb.png)
![Screenshot_20210729-181605](https://user-images.githubusercontent.com/24504742/127475914-b5d6fc75-2d1e-4205-8f1c-4cdf4555e1df.png)

The second screenshot is taken on android 10. As you can see the navigation bar blocked the content behind it.
The first screenshot is what i want and what it looks like on android 11.

It seems that in the [code](https://github.com/robertodoering/harpy/blob/7ff286068a7ba91e92bdf85d519082ec70294977/lib/components/settings/theme_selection/bloc/theme_bloc.dart#L45) this feature is limited to android 11 and above.
I am not familiar with dart maybe I got it wrong. Sorry for bad english.

**Comment by robertodoering** [comment]
You are right, only android 11 was using the transparent navigation bar color and to be honest I don't remember why I limited it to android 11.
The flag used in the android runner to enable the app to draw behind the navigation bar was introduced in 4.1, so there was no reason limit the transparency to android 11.

Thanks a lot for making me aware of this!
I have removed the condition now and tested it with android 8, 9, 10 and 11 and it seems to work fine.
This will be released with the next update :)

https://github.com/robertodoering/harpy/commit/9022e3bf05b1c5d228db4aa4dca222412548cc0d#diff-53887454abe13f4b00da22db44516e239de0ae8eaa1fca00502bd83c432a4ee0

**Comment by Steve-Mr** [comment]
Sorry to bother you again.
The version 0.6.12 works on android 10 well except some little bugs.
The navigation bar is transparent at first, but after restarting the app (swipe away from recents sreen) it is no longer transparent, you can check the screen record
However, the swan theme doesn't have this problem and change the theme to swan can make the navigation bar transparent again.
I’m sorry to disturb you many times because of an old system, thank you for your excellent work！

https://user-images.githubusercontent.com/24504742/128893585-61f708a5-a07c-4fb3-bf6f-287fecf80ba3.mp4

**Comment by robertodoering** [comment]
Hey @Steve-Mr,
Thanks for letting me know about the issue!

I'll have a look soon and see if I can fix it.
